apps/extension/src/routes/popup/settings/settings-clear-cache.tsx

```diff
@@ -11,15 +11,14 @@ import { SettingsScreen } from './settings-screen';
 const useCacheClear = () => {
   const navigate = usePopupNav();
   const [loading, setLoading] = useState(false);
+  const setFullSyncHeight = useStore(state => state.network.setFullSyncHeight);
 
   const handleCacheClear = () => {
     setLoading(true);
 
     void (async function () {
+      await setFullSyncHeight(0);
```

Comment:
question: Does a height synced of `0` mean that the genesis state has been synced? I actually think the data structure is more like:

```
fullSyncHeight = undefined --> no syncing has been completed
fullSyncHeight = 0 --> genesis state synced
fullSyncHeight = <number> --> that block number has been synced
```

Can you validate this? May help checking the logic of the loading bar to see what text shows under which condition.

Reply:
The are no checks like `if (typeof fullSyncHeight === 'undefined'), only `!fullSyncHeight`. This way both `0` and `undefined` mean loading, `number` means syncing, and `latestKnownBlockHeight - fullSyncHeight <= 10` means synced

Reply:
Updated zeros to `undefined`